sign-in-with-esignet/package.json

```diff
@@ -0,0 +1,108 @@
+{
+  "name": "sign-in-with-esignet",
+  "version": "0.0.1",
+  "description": "A react component for rendering “Sign in with..” button on mosip client’s web page for oidc integration.",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "homepage": "https://docs.esignet.io",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mosip/esignet-plugins/sign-in-with-esignet"
+  },
+  "files": [
+    "dist",
+    "README.md"
+  ],
+  "keywords": [
+    "mosip",
+    "button",
+    "login",
+    "signin",
+    "signup",
+    "oidc",
+    "openidconnect",
+    "esignet"
+  ],
+  "license": "MIT",
```

Comment:
MPL 2.0? or MIT? our repo states its MPL 2.0

Reply:
license updated to MPL 2.0